keras_nlp/src/utils/preset_utils.py

```diff
@@ -704,35 +705,54 @@ def load_tokenizer(self, cls, **kwargs):
         tokenizer.load_preset_assets(self.preset)
         return tokenizer
 
-    def load_task(self, cls, load_weights, **kwargs):
+    def load_task(self, cls, load_weights, load_task, **kwargs):
```

Comment:
Isn't this a little bit confusing that the function is called `load_task` and the new flag is also called `load_task`? 
If the `load_task` flag is false, it only loads the backbone so it's not loading the task anymore so the function name becomes confusing 🤔

Reply:
Yeah good call! This function isn't exposed at least but I'll think.

Reply:
Maybe `load_task_extras` would be a better name for this.

Reply:
Trying this out.